### ISSUE TITLE
INSTALL_PARSE_FAILED_NOT_APK

### ISSUE THREAD

**Issue by toas-koas** [MuntashirAkon/AppManager#900]
If I install an app via the aurora store, then this message appears

`10-22 10:45:47.024 D/ACCESSIBILITY(29832): INSTALL_PARSE_FAILED_NOT_APK: Failed to parse /data/app/vmdl1153403362.tmp/file_16664262404405861766839290669084.cached: Failed to load asset path /data/app/vmdl1153403362.tmp/file_16664262404405861766839290669084.cached]; ContentDescription: null; ItemCount: -1; CurrentItemIndex: -1; Enabled: true; Password: false; Checked: false; FullScreen: false; Scrollable: false; BeforeText: null; FromIndex: -1; ToIndex: -1; ScrollX: 0; ScrollY: 0; MaxScrollX: 0; MaxScrollY: 0; ScrollDeltaX: -1; ScrollDeltaY: -1; AddedCount: -1; RemovedCount: -1; ParcelableData: null ]; recordCount: 0`

**Comment by MuntashirAkon** [comment]
Closed because the issue doesn't follow any template.
